### ISSUE TITLE
Support tools for CiviForm Admins: allow support for user accounts

### ISSUE THREAD

**Issue by msprenke** [civiform/civiform#3434]

**Comment by nbeloglazov** [comment]
Given that civiform doesn't implement authentication itself and instead relies on external auth providers - the issues like resetting passwords need to be handled in those external auth providers. For Seattle that would be https://login.seattle.gov/ So team who owns or configures login.seattle.gov will be the one to help. 

I'm curious, "forgot password" links on https://login.seattle.gov/ are not sufficiently helpful for applicants and they end up contacting tech support?

**Comment by msprenke** [comment]
You are correct about the direction this needs to take.  I contacted the support team and was given access to a tool to view users login info, emails, and reset passwords.

The forgot password link is sufficient in most cases however if a user doesn't receive the activation email, there is nothing anyone can do without getting the help of a developer.  I now have the ability to delete the account so they can recreate it.  They are researching a resend activation email function.

So while this doesnt live in the CiviForm development circle, it remains something I'm tracking for Seattle.

**Comment by msprenke** [comment]
Login admin tool has been provided
